lib/Dialect/FIRRTL/Transforms/LowerClasses.cpp

```diff
@@ -764,10 +767,16 @@ std::unique_ptr<mlir::Pass> circt::firrtl::createLowerClassesPass() {
   return std::make_unique<LowerClassesPass>();
 }
 
+// Predicate to check if a module-like needs a Class to be created.
+bool LowerClassesPass::shouldCreateClass(StringAttr modName) {
+  // Return a memoized result.
+  return shouldCreateClassMemo.at(modName);
+}
+
 // Predicate to check if a module-like needs a Class to be created.
 bool LowerClassesPass::shouldCreateClass(FModuleLike moduleLike) {
```

Comment:
Any specific reason not to change this predicate, rather than defining a new predicate with a different signature?

Reply:
This was an oversight, updated the signature .